invenio_notifications/config.py

```diff
@@ -57,3 +57,6 @@
 This should be set higher up in the module hierarchy (e.g. invenio-app-rdm), as
 this module does not have knowledge of the settings view.
 """
+
+NOTIFICATIONS_MAIL_REPLY_TO = None
+"""Reply to mail address for notifications."""
```

Comment:
maybe add a comment that if None what would be the mail sender? Also, after testing, maybe add here the comment about `MAIL_DEFAULT_SENDER`?

Reply:
The mail sender is always `MAIL_DEFAULT_SENDER`. This is only for the `reply to` attribute of the email. I will put some explanations in the comment.